day3/main.go

```diff
@@ -0,0 +1,71 @@
+package day3
+
+import (
+	_ "embed"
+	"strings"
+)
+
+//go:embed input.txt
+var input string
```

Comment:
Initial reaction: "What sorcery is this!"
Also: "Niiiiice"

Reply:
:D

Reply:
yeah - really clean way to embed files into a package